javascript/javascript2/week1/js2week1.js

```diff
@@ -0,0 +1,66 @@
+// Problem solving cardio
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+
+function findShorty() {
+    const shortestWord = danishWords[0].length
+    for (let i = 0; i < danishWords.length; i++) {
+        const word = danishWords[i].length
+        if (word < shortestWord) {
+            word = shortestWord
+        }
+        return shortestWord
```

Comment:
When you return inside of for, it's going to finish the iteration and function, so the result will always be your first element length.

Reply:
also, we need to show a word, but you are showing the length of the word, so I think maybe you could do it like
```
function findShorty() {
    let shortestWord = danishWords[0]
    for (let i = 0; i < danishWords.length; i++) {
        if (danishWords[i].length < shortestWord.length) {
            shortestWord = danishWords[i]
        }
    }
    return shortestWord
}
```